code/processes/segmentedtickerplant.q

```diff
@@ -71,6 +76,17 @@ setup:{[batch]
   if[not system "t";.lg.o[`timer;"defaulting timer to 1000ms"];system"t 1000"];
  };
 
+// load subfilters for rdb
+loadsubfilters:{[]
+	.sub.filterparams:@[{1!("S**";enlist",")0: x};.rdb.subcsv;{.lg.e[`loadsubfilters;"Failed to load .rdb.subcsv with error: ",x]}];
+	.rdb.subscribeto:raze value flip key .sub.filterparams;
+	.rdb.subscribesyms:.sub.filterparams;}
+
+// load config file if it exists
+// just giving some sample config
+tpconfig: ([] table:`trade`trade`quote`quote; segmentID:1 2 1 2; wcRef:`wcA`wcB`wcC`wcD; filters:`AAPL`GOOG`AAPL`GOOG)
+procmap: (enlist `rdb1)!(enlist 1)
+
```

Comment:
As above, all this needs to be removed. Be careful about what you commit. This is all temp code and not required.

Reply:
removing this, will also look at error trapping for when there is no config